Dockerfile

```
@@ -1,11 +1,11 @@
-FROM mcr.microsoft.com/dotnet/sdk:6.0.300-alpine3.15 AS build
+FROM mcr.microsoft.com/dotnet/sdk:6.0.400-alpine3.15 AS build
```

Lets upgrade to the latest and greatest
```suggestion
FROM mcr.microsoft.com/dotnet/sdk:6.0.401-alpine3.16 AS build
```